2025/build-tooling/slides.md

```diff
@@ -16,25 +16,126 @@ is: feedback
 
 ---
 
-# Slide
+# Audience
 
-Slide content
+- Developers looking to add build step to their apps
+- Developers looking to learn more about the capabilities of the build tools
+- Developers interested in how Esri's teams are leveraging modern build tools
 
 
 ---
-layout: intro
+
+# Agenda
+
+TODO: create agenda based on the presentation outline
+
+
+
+---
+
+# Benefits of using build tools
+
+- Ease of using external libraries
+- Can import/export between files
+- Live reload
+- Better performance - minification and bundle splitting
+- Can use modern JS syntax, which will get downcompiled as needed
+- Can build complete offline apps with local assets
+- TypeScript, IntelliSence
+  - TODO: Can borrow material from my previous year's TypeScript Benefits list https://github.com/maxpatiiuk/esri-dev-summit-presentations/blob/main/2024/calcite-react/index.md#typescript-
+  - show example of bugs TypeScript can easily catch?
+  - or show autocomplete for Maps SDK APIs?
+- more customizability, control, support for advanced use cases
+
+TODO: split into multiple slides or reduce. Move TypeScript out further down?
```

Comment:
I like the idea of this slide acting as an overview for the bulk of presentation -- "here are the benefits we are going to cover, don't worry if you don't know what they mean, we'll break them down and cover them in demos"

And for discussion: Are there any here that we really want to deep-dive into? Eg should we spend 10 minutes talking about and demoing minification / bundle splitting / tree shaking